content/docs/setup/kubernetes/install/multicluster/gateways/index.md

```diff
@@ -8,8 +8,8 @@ keywords: [kubernetes,multicluster,federation,gateway]
 ---
 
 Follow this flow to install an Istio [multicluster service mesh](/docs/concepts/multicluster-deployments/)
-where the Kubernetes cluster services and the applications in each cluster
-are limited to remote communication using gateway IPs.
+when a Kubernetes cluster's services should use the gateway to communicate
+to remote services.
```

Comment:
How about:
```
when services in a Kubernetes cluster use gateways to communicate with services in other clusters.
```

Reply:
good word-smithing frank - thanks for fixing the problem there :)